packages/babel-code-frame/src/index.ts

```diff
@@ -3,15 +3,37 @@ import highlight, { shouldHighlight, getChalk } from "@babel/highlight";
 let deprecationWarningShown = false;
 
 type Location = {
-  column: number,
-  line: number,
+  column: number;
+  line: number;
 };
 
 type NodeLocation = {
-  end: Location,
-  start: Location,
+  end?: Location;
+  start: Location;
 };
 
+export interface BabelCodeFrameOptions {
```

Comment:
Since we will be hopefully exporting this to our users in the future, I'd prefer to call it just `Options` (since `BabelCodeFrame` is implied by the package name).

Reply:
took it from definitely typed - https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/babel__code-frame/index.d.ts#L17

not sure what option we should choose, my guess was that it would be preferable if we can keep it compatible with types people are already using (however not everything is currently compatible - was thinking, to do it as separate step after everything is migrated to ts)

Reply:
We can export it as `Options` and then add something like
```js
/** @deprecated Use `Options` instead */
export { Options as BabelCodeFrameOptions };
```
if compatibility with `.d.ts` is a concern (it has never been an official `.d.ts` source for Babel, so it shouldn't restrict our migration).

Reply:
Agree, not overly concerned with whats on `DefinitelyTyped`.

Reply:
lets rename it for now to just `Options`, and maybe latter before starting to publish types for this package - to align it with definitely typed